packages/react-instantsearch/src/ui/__tests__/Pagination.test.tsx

```diff
@@ -32,10 +32,10 @@ describe('Pagination', () => {
         nextPageItemText: '›',
         lastPageItemText: '››',
         pageItemText: ({ currentPage }) => String(currentPage),
-        firstPageItemAriaLabel: 'First',
-        previousPageItemAriaLabel: 'Previous',
-        nextPageItemAriaLabel: 'Next',
-        lastPageItemAriaLabel: 'Last',
+        firstPageItemAriaLabel: 'First Page',
```

Comment:
why not other changes in React InstantSearch?

Reply:
reverting this, as i hadn't caught that it was in wrong package. good catch! thank you